aspnetcore/mvc/models/validation.md

```diff
@@ -140,7 +140,7 @@ The tag helpers above render the HTML below. Notice that the `data-` attributes
 </form>
 ```
 
-Client-side validation prevents submission until the form is valid. The Submit button runs JavaScript that either submits the form or displays error messages.
+Thus, client-side validation prevents submission until the form is valid. The Submit button runs JavaScript that either submits the form or displays error messages.
```

Comment:
~Thus,~

Reply:
Thought it added a nice oomph, but no worries :) I'll wait for @rachelappel's feedback before making any more commits.